Codecademy/discourse-codebytes-plugin
# Remove unused engine and routes

## Overview
Removes the Rails engine in lib/code-bytes/engine.rb, which we weren't using. Also has some misc cleanup in plugin.rb and elsewhere.

### PR Checklist
- [x] Related to JIRA ticket: [REACH-963](https://codecademy.atlassian.net/browse/REACH-963)

### Testing Instructions
1. Run Discourse locally. Head to http://localhost:3000/
2. Check that plugin still loads: toolbar button appears and iframes are rendered.
